code/__DEFINES/fov.dm

```diff
@@ -0,0 +1,11 @@
+/// Field of vision defines.
+#define FOV_90_DEGREES 90
+#define FOV_180_DEGREES 180
+#define FOV_270_DEGREES 270
```

Comment:
Why are these defines? When will they change? Their value is already embedded in their name

Reply:
These exist so people don't use a value that doesn't have a corresponding mask in the .dmi file

Reply:
Where is a define for 0 degrees, then